docker-compose.yaml

```diff
@@ -53,9 +53,12 @@ services:
     command:
       - '/usr/bin/env'
       - 'python3'
-      - '/home/kernelci/pipeline/kcidb.py'
+      - '/home/kernelci/pipeline/send_kcidb.py'
       - '--settings=/home/kernelci/config/kernelci.conf'
       - 'run'
+    environment:
+      - GOOGLE_APPLICATION_CREDENTIALS=/home/kernelci/data/kcidb/kernelci-production-admin.json
```

Comment:
This overrides whatever value was set in the `.env` file, so please drop it.  I'm getting this error:

http://staging.kernelci.org:9088/container/142d8327dffb

```
today at 11:36:02 AMgoogle.auth.exceptions.DefaultCredentialsError: File /home/kernelci/data/kcidb/kernelci-production-admin.json was not found.
today at 11:36:22 AMContainer stopped
```
even though:
```
$ grep CREDENTIALS .env
GOOGLE_APPLICATION_CREDENTIALS=data/kcidb/kernelci-production-ci-kernelci.json
```

Reply:
I'm putting a symlink in place with the hard-coded name to see if the rest works on staging, but this should be fixed.

Reply:
OK that seems to have worked:
http://staging.kernelci.org:9088/container/a79fe8686bc5
```
today at 11:44:38 AMListening for events... 
today at 11:44:38 AMPress Ctrl-C to stop.
today at 11:45:31 AMPrinting node for {'_id': '62176fa64cfcc5a6431d4b54', 'kind': 'node', 'name': 'checkout', 'revision': {'tree': 'kernelci', 'url': 'https://github.com/kernelci/linux.git', 'branch': 'staging-mainline', 'commit': 'fc5b1486b7664213f96da97ad8515b36d9875bb9', 'describe': 'staging-mainline-20220223.0'}, 'parent': None, 'status': True, 'artifacts': {'tarball': 'http://staging.kernelci.org:9080/linux-kernelci-staging-mainline-staging-mainline-20220223.0.tar.gz'}, 'created': '2022-02-24T11:44:38.120000', 'updated': '2022-02-24T11:45:31.697000'}
today at 11:45:31 AMSubmitting node to KCIDB
```

Reply:
I also removed the symlink which was a workaround for the hard-coded file name and that's working too now.